src/Test/HttpClient/MockHttpAdapter.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace FOS\HttpCache\Test\HttpClient;
+
+use Http\Adapter\Exception\MultiHttpAdapterException;
+use Http\Adapter\HttpAdapter;
+use GuzzleHttp\Psr7\Response;
+use Psr\Http\Message\RequestInterface;
+
+class MockHttpAdapter implements HttpAdapter
```

Comment:
Docblock added. I propose to merge this PR first, then move the mock to php-http.